crates/rooch-indexer/src/models/transactions.rs

```diff
@@ -82,22 +69,33 @@ impl From<IndexedTransaction> for StoredTransaction {
     }
 }
 
-impl StoredTransaction {
-    pub fn try_into_transaction_with_info(self) -> Result<TransactionWithInfo, anyhow::Error> {
-        let transaction = LedgerTransaction::decode(&self.transaction_raw)?;
+impl TryFrom<StoredTransaction> for IndexerTransaction {
+    type Error = anyhow::Error;
 
-        let status: KeptVMStatus = serde_json::from_str(self.status.as_str())?;
-        let execution_info = TransactionExecutionInfo {
-            tx_hash: H256::from_str(self.tx_hash.as_str())?,
-            state_root: H256::from_str(self.state_root.as_str())?,
-            size: self.size as u64,
-            event_root: H256::from_str(self.event_root.as_str())?,
-            gas_used: self.gas_used as u64,
-            status,
+    fn try_from(transaction: StoredTransaction) -> Result<Self, Self::Error> {
+        let sender = RoochAddress::from_hex_literal(transaction.sender.as_str())?;
+        let tx_hash = H256::from_str(transaction.tx_hash.as_str())?;
+        let tx_accumulator_root = H256::from_str(transaction.tx_accumulator_root.as_str())?;
+        let state_root = H256::from_str(transaction.state_root.as_str())?;
+        let event_root = H256::from_str(transaction.event_root.as_str())?;
+
+        let indexer_transaction = IndexerTransaction {
+            tx_hash,
+            tx_order: transaction.tx_order as u64,
+            sequence_number: transaction.sequence_number as u64,
+            sender,
+            action: None,
```

Comment:
Should we remove the action field from IndexerTransaction? If we do not store it.